test/image/Dockerfile

```diff
@@ -32,12 +32,12 @@ RUN curl -sSLO https://github.com/bats-core/bats-core/archive/v${BATS_VERSION}.t
     bash bats-core-${BATS_VERSION}/install.sh /usr/local
 
 # Install ORAS
-RUN curl -SsLO https://github.com/oras-project/oras/releases/download/v${ORAS_VERSION}/oras_${ORAS_VERSION}_linux_amd64.tar.gz && \
+RUN curl -SsLO https://github.com/oras-project/oras/releases/download/v${ORAS_VERSION}/oras_${ORAS_VERSION}_linux_${arch}.tar.gz && \
```

Comment:
can we use `TARGETARCH` here? https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/

Reply:
thanks for the advice.
I search the web , [docker automatic-platform-args-in-the-global-scope](https://docs.docker.com/engine/reference/builder/#automatic-platform-args-in-the-global-scope).
It says 
```
Automatic platform ARGs in the global scope
This feature is only available when using the [BuildKit](https://docs.docker.com/build/buildkit/) backend.
```

perhaps in docker build mod , it will no works for ..